### ISSUE TITLE
Zwave Fibaro FS-223 Switch 2 doesn't work reliably

### ISSUE THREAD

**Issue by minchik** [OpenZWave/open-zwave#1280]
I have Aeotec ZW090 Z-Stick Gen5 and Fibaro System FGS223 Double Relay. It has two switches. The first one works great. But I have to toggle second switch twice to work properly.
If I turn on it at first time, it actually turns on the light but web interface doesn't recognize that it was turned on. If I turn on it again, web interface understands new status. Same thing for turning off.
This is not just my issue, it is actually wide spread across Home Assistant users. At first I though  that this is home-assistant bug, but then I repeated It using OpenZWave Control Panel.

I'v recorded [screen cast](https://youtu.be/FWz2PvbGtak)
My [zwave-log.txt](https://github.com/OpenZWave/open-zwave/files/1166901/zwave-log.txt)
[Home Assistant issue](https://github.com/home-assistant/home-assistant/issues/7076)

**Comment by nechry** [comment]
The controller need to be with instance 1 in Lifeline group

**Comment by minchik** [comment]
Ok, I'm newbie in z-wave and openzwave particularly. Do you mean this:
![openzwave_control_panel_2017-07-22_12-44-36](https://user-images.githubusercontent.com/2690703/28490171-de3e0938-6edb-11e7-804d-c3768cc4a6a8.jpg)

**Comment by nechry** [comment]
Yes you must have 1.1 not 1

**Comment by minchik** [comment]
But there is no such option
<img width="1643" alt="screen shot 2017-07-22 at 20 33 40" src="https://user-images.githubusercontent.com/2690703/28493313-4701c09a-6f1d-11e7-8e50-d4b03c1cfa09.png">

**Comment by nechry** [comment]
Its suppose to be automatically perform during the inclusion process. It's a fresh new included device? I don't know how to force the instance 1 with your tool, In jeedom gateway its always available to choose  instance 1 or not.

**Comment by minchik** [comment]
My controller with nodeId 1 is in Lifeline group of fibaro switch. I think that the issue is not in associations, as first channel of the fibaro switch works perfect. The problem is in the second channel only.

Could you read **OpenHAB Integration** part of that article https://www.nilsschneider.de/wp/2016/09/11/fibaro-double-switch-2-fgs-223-und-openhab/ I think this is my case

**Comment by nechry** [comment]
I have this device and i fully tested. Without specifing the instance 1 I got same bad behavior with the device. With 1.1 (controller and instance) the 2 end points work as expected

**Comment by minchik** [comment]
May be my answer is stupid 😄 , but this device doesn't have group 1.1 in associations. 

http://manuals.fibaro.com/switch-2/

**Comment by nechry** [comment]
1.1  representing controller.instance, usually node 1 is the controller so 1.1 in group Lifeline

**Comment by minchik** [comment]
You mean 1.1 should be there?

<img width="1643" alt="11" src="https://user-images.githubusercontent.com/2690703/28510229-cc23eb12-704f-11e7-8fc3-23efd8faabd3.png">

**Comment by minchik** [comment]
By the way, all the screenshots are taken from [open-zwave-control-panel](https://github.com/OpenZWave/open-zwave-control-panel) which, I think, is developed by the owners of open z-wave project

**Comment by nechry** [comment]
In jeedom it 's possible to do this. A change is required is ozcp to allow choose instance 1 of the controller.

**Comment by nechry** [comment]
ensure your device configuration files are up to date in your ozcp. Exclude the node and include it again. Is supposed to be automatically get the 1.1

**Comment by nechry** [comment]
see: https://github.com/OpenZWave/open-zwave/blob/master/config/fibaro/fgs223.xml#L267
ForceInstances="true" is the key

**Comment by nechry** [comment]
small note: check my commit
https://github.com/OpenZWave/open-zwave/commit/d39a11ed43856437f719088eb4850d74535ee36e#diff-be80d77d3784d4c46d8a2cacd2acbe5b
your device is included in secure ? if yes me be is your issue

**Comment by minchik** [comment]
No my device is not included in secure mode. So I have to include it securely?

**Comment by minchik** [comment]
If that doesn't help, is it possible to install  jeedom, change this option and than use my software? Is that config parameter stored directly in relay?

**Comment by nechry** [comment]
I got a issue if is included securely, so no stay in normal way.
Check you local fibaro/fgs223.xml file and ensure is same a repo file.

Yes you can install jeedom with zwave plugin do to this, your network topology is stored in your dongle and the association group and configuration in your device. If you try jeedom you will keep it... lol!

**Comment by nechry** [comment]
if you wan try jeedom, start from a fresh Debian 8 install
`
wget https://raw.githubusercontent.com/jeedom/core/stable/install/install.sh
chmod +x install.sh
./install.sh
`

**Comment by minchik** [comment]
Now I know what was my issue. I've added a relay to the z-wave network as described in the user manual of the Aeotec stick. I have pulled out the stick from the computer and then pressed the button on the stick. But this process is error prone. All devices should be added to z-wave network directly from ozcp.
@nechry Thank you very much!!!
 I think this issue can be closed. Will you close it or I should close it myself?
